### PR TITLE
ELK-Risk-Based

### PR DESCRIPTION
**Pull request by Avarjana**
### Proposed changes in this pull request
- Add ELK-Risk-Based script
- Deprecate IS Analytics based risk score script

### When should this PR be merged
Issue: wso2/product-is#13523
Related PR: wso2-extensions/identity-conditional-auth-functions#98

### CODE REVIEW

File: features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/elasticRiskScore.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  "helpLink": "https://is.docs.wso2.com/en/latest/learn/configuring-risk-based-adaptive-authentication/",
+  "code": [
+    "// Define conditional authentication by risk score value calculated from ELK analytics.",
+    "",
+    "var onLoginRequest = function(context) {",
+    "    executeStep(1, {",
+    "        onSuccess: function (context) {",
+    "            var username = context.currentKnownSubject.username;",
+    "            callElastic('https://localhost:9200', {'username': username, 'elasticUsername':'<username>', 'elasticPassword':'<password>'}, {",
```
Comment (ashensw) [inline features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/elasticRiskScore.json:42]: Don't we have a way to configure the ELK URL?

Comment (Avarjana) [reply features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/elasticRiskScore.json:42]: Can configuring Elasticsearch URL in deployment.toml address this issue @ashensw?
```
[analytics.elk]
enable=true
elastic_url="https://localhost:9200"
```

Comment (Avarjana) [reply features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/elasticRiskScore.json]: Resolved with governance connector config.

**Comment by Avarjana** [comment]
I have made a mistake and messed up git history. I will create a new PR and close this one. Sorry for the inconvenience.

**Pull request closed without merging**